src/WaterLily.jl

```diff
@@ -132,6 +132,15 @@ function restart_sim! end
 # export
 export restart_sim!
 
+#default Plots functions
+function flood end
+function addbody end
+function body_plot! end
+function sim_gif! end
+function plot_logger end
+# export
+export flood,addbody,body_plot!,sim_gif!,plot_logger
+
```

Comment:
at some point, we will have a lot of those. Could we not hide them in an `ext/default.jl` file?

Reply:
Since these are `Plots.jl` related, I would using an extension for Plots, so that whenever it's loaded together with WaterLily, all of this function will be compiled too.

Reply:
And the problem is not really compiling those, but loading Plots. I think it is fine to have them in a single file, but if it gets two crowded we can also have separate files in `/ext` that are `include`ed in the extension file.